README.md

```diff
@@ -1,128 +1,147 @@
 # PythonBasics
 
-## To launch:
-```
-cd ~/git/.../PythonBasics/script/folder
+## To run
+
+```shell
+git clone https://github.com/AlexDesBen/PythonBasics.git
+cd PythonBasics/script/HelloWorld/
+python Main.py
+cd ../ExampleNumpy/
 python Main.py
 ```
+
 ## To know
 
 ### Importing libraries
-```
+
+```python
 import library
 import library as nickname
 from library import module
 from library import module as nickname
 ```
-With a library you can use all the modules and sub modules like this
-```
+
+With a library you can use all the modules and sub modules like this :
+
+```python
```

Comment:
Thanks for this I did not know about this feature

Reply:
Markdown is pretty useful, you'll discover many many other things. Look up **Markdown GFM** 😜